src/analog/message.ts

```diff
@@ -144,6 +142,29 @@ export const fromBase64 = ( data: string ) => (
     fromBytes(getBytesFromBase64(data))
 );
 
+export const toMessage = ( commands: Array<Command> ): IMessage => {
+    const commandsBinary = commands.map(command => ({
+        command,
+        ...command.toBinary()
+    }));
+
+    const body = commandsBinary.reduce(
```

Comment:
just an idea

take a look at Array.prototype.flat

Reply:
[[1], [2]].flat() will give us [1, 2]
but we have
[{buffer: [1]}, {buffer: [2]}]

Reply:
`[{buffer: [1]}, {buffer: [2]}].map(item => ...item).flat()`

?

Reply:
fixed